tools/esp32/Config.mk

```diff
@@ -23,8 +23,8 @@
 # archive.  These replace the default definitions at tools/Config.mk
 
 ifdef BLOBDIR
```

Comment:
BLOBDIR could be renamed to something more intuitive.
Furthermore, it may be interesting to document what `BLOBDIR` intends to achieve.

Reply:
Do you have a better name?  I couldn't come up with something better, It's just a directory that contains blobs :)

Reply:
I feel that calling it a BLOB gives the wrong impression that those are closed-source, which I think is not the case for the second stage bootloader and the partition table, right?
Maybe `ESPTOOL_BINDIR`, but I'm not also convinced about it, because BINDIR seems to relate to "install dir"...
It is no wonder that "naming things" is one of the most difficult things in computer science hehehe

Reply:
No, they are not closed source.  They are part of IDF.  Let's go with `ESPTOOL_BINDIR`, at least this is somewhat consistent with the other parameter (`ESPTOOL_PORT`)